ZZ_testing_channels.md

```diff
@@ -0,0 +1,163 @@
+Testing Phoenix Channels will help ensure that your code continues to work
+as you make changes to your application. It helps guide development and
+ensure there is no breakage later on down the road.
+
+Phoenix has built in helper functions in `Phoenix.Channel.Test` that help
+make testing Channels easy.
+
+### Testing `join` authorization
```

Comment:
Maybe we could give readers a little intro to what we're about to do here - ease them into it a bit - since this is the first encounter many might have had with testing channels.